IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -308,9 +416,39 @@ - (nullable instancetype)initWithGroup:(nullable NSString *)keychainGroup
 
         if (!self.keychainGroup)
         {
+            if (error)
+            {
+                *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInternal, @"Failed to set keychain access group.", nil, nil, nil, nil, nil);
+                MSID_LOG_WITH_CTX(MSIDLogLevelError, nil, @"Failed to set keychain access group.");
```

Comment:
nit: please produce log regardless if error is passed or not

Reply:
Addressed.